src/features/views/components/ViewDataTable/columnTypes/SurveyOptionColumnType.tsx

```diff
@@ -0,0 +1,125 @@
+  implements IColumnType<SurveyOptionViewColumn, SurveyOptionViewCell>
+{
+  cellToString(cell: SurveyOptionViewCell): string {
+    const pickedThisOption = cell.filter((submission) => submission.selected);
+    return pickedThisOption.length ? pickedThisOption[0].submitted : '';
+  }
+
+  getColDef(): Omit<GridColDef<SurveyOptionViewColumn>, 'field'> {
+    return {
+      headerAlign: 'center',
+      renderCell: (params) => {
+        return <Cell cell={params.value} />;
+      },
+    };
+  }
+
+  getSearchableStrings(): string[] {
+    return [];
+  }
+}
+
+const Cell: FC<{ cell: SurveyOptionViewCell }> = ({ cell }) => {
+  const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
+  const { openPane } = usePanes();
+  const { orgId } = useRouter().query;
+
+  if (!cell.length) {
+    return null;
+  }
+
+  const hasPickedThisOption = !!cell.filter((submission) => submission.selected)
+    .length;
+
+  if (!hasPickedThisOption) {
+    return null;
+  }
+
+  const sorted = cell.concat().sort((sub0, sub1) => {
+    const d0 = new Date(sub0.submitted);
+    const d1 = new Date(sub1.submitted);
+    return d1.getTime() - d0.getTime();
+  });
+
+  const mostRecent = sorted[0];
+
+  return (
+    <Box
+      alignItems="center"
+      bgcolor={mostRecent.selected ? theme.palette.success.light : ''}
+      display="flex"
+      height="100%"
+      justifyContent="center"
+      onMouseOut={() => setAnchorEl(null)}
+      onMouseOver={(ev) => setAnchorEl(ev.currentTarget)}
+      width="100%"
+    >
+      {mostRecent.selected ? (
+        <Check sx={{ opacity: '0.5' }} />
+      ) : (
+        <History color="secondary" />
+      )}
+      <ViewSurveySubmissionPreview
+        anchorEl={anchorEl}
+        onOpenSubmission={(id) => {
+          openPane({
+            render() {
+              return (
+                <SurveySubmissionPane
+                  id={id}
+                  orgId={parseInt(orgId as string)}
+                />
+              );
+            },
+            width: 400,
+          });
+        }}
+        submissions={sorted.map((sub, index) => {
+          let matchingContent: JSX.Element | null = null;
+
+          if (sub.selected) {
+            matchingContent = (
+              <Box alignItems="center" display="flex">
+                <Check sx={{ paddingRight: 1 }} />
+                <FormattedMessage id="misc.views.surveyOptionCell.selected" />
+              </Box>
+            );
+          } else if (index === 0 && !sub.selected) {
+            matchingContent = (
+              <Typography color="secondary" sx={{ fontStyle: 'italic' }}>
+                <FormattedMessage id="misc.views.surveyOptionCell.notSelected" />
+              </Typography>
+            );
+          }
+
+          return {
+            id: sub.submission_id,
+            matchingContent: matchingContent,
+            submitted: sub.submitted.toString(),
+          };
+        })}
```

Comment:
Nice solution! I'm so excited to see you tackle stuff like this with such ease and grace even with very little instructions nowadays (but also sad because I miss mentoring you).